contributing.md

```diff
@@ -90,6 +90,13 @@ features.add({
 });
 ```
 
+## Requirements
+
+[Node](https://nodejs.org/en/download/) version 12 or up is required. You can check your current version with:
```

Comment:
Node => Node.js

or up => or later


I would drop the instructions on how to check the version. That’s expected knowledge.